### PR TITLE
Rename to @openzeppelin/contracts-ethereum-package

### PR DESCRIPTION
**Pull request by spalladino**
I'm opening this PR against a new `@openzeppelin` branch, since I'm not sure if we want to merge this into master right away. I don't see any reason not to though!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 # <img src="logo.png" alt="OpenZeppelin" width="400px">
 
-# OpenZeppelin EVM Package
+# OpenZeppelin Ethereum Package
```
Comment (nventuro) [inline README.md]: Shouldn't this be 'OpenZeppelin Contracts Ethereum Package'?

**Comment by nventuro** [comment]
Not sure we want to merge into master yet since this includes links to repos that don't yet exist, to the old zos docsite, etc.

File: package.json [hunk 2]
```diff
@@ -1,13 +1,12 @@
 {
-  "name": "openzeppelin-eth",
-  "version": "2.2.0",
+  "name": "@openzeppelin/contracts-ethereum-package",
+  "version": "2.2.1",
```
Comment (frangio) [inline package.json]: Shouldn't this be a release candidate, if the dependencies we're using are also release candidates?

**Comment by frangio** [comment]
LGTM. I have no idea why the CI is failing, but it was failing before too. :slightly_smiling_face: 

I'm contacting Travis support to migrate the repo to from travis-ci.org to travis-ci.com.

**Comment by spalladino** [comment]
> Not sure we want to merge into master yet since this includes links to repos that don't yet exist, to the old zos docsite, etc.

@nico I'd merge either way, so we ensure we keep working on the renamed version.

**Pull request merged**